qualitycontrol.sh

```diff
@@ -58,7 +58,18 @@ JOBS=$((${JOBS:-1}*3/5))
 cmake --build . -- ${JOBS:+-j$JOBS} install
 
 if [[ $ALIBUILD_O2_TESTS ]]; then
-  make test
+  TESTERR=
+  ctest --output-on-failure ${JOBS+-j $JOBS} || TESTERR=$?
+  # Display additional logs for tests that timed out in a non-fatal way
+  set +x
+  for LOG in test_logs/*.nonfatal; do
```

Comment:
hum, as far as I can tell, this will be ineffective (at best), as the `test_logs` structure is not a ctest feature, but one the O2 `tests-wrapper.sh` script.
Besides, that's not need in QC a there's no such thing as a non-fatal test in QC, right ?

Reply:
correct, I went too fast on the copy-paste road.